lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1878,8 +1881,31 @@ signTransaction ctx (ApiT wid) body = do
     let pwd = coerce $ body ^. #passphrase . #getApiT
     let sealedTx = body ^. #transaction . #getApiT
 
-    sealedTx' <- withWorkerCtx ctx wid liftE liftE $ \wrk ->
-        liftHandler $ W.signTransaction wrk wid pwd sealedTx
+    sealedTx' <- withWorkerCtx ctx wid liftE liftE $ \wrk -> liftHandler $ do
+        let
+            db = wrk ^. W.dbLayer @IO @s @k
```

Comment:
We have those parts in almost all places in `where`. Shouldn't we have here also (just for consistency)?

Reply:
It won't work in this case, because `wrk` is not an argument to the outer function, `wrk` won't be in scope in the `where` block.